mobile-app/app/screens/AppNavigator/screens/Dex/components/PoolPairCards/PoolPairTextSectionV2.tsx

```diff
@@ -27,8 +27,8 @@ export function PoolPairIconV2 (props: {
   const IconB = props.symbolB === 'DFI' ? getNativeIcon('_UTXO') : getNativeIcon(props.symbolB)
   return (
     <>
-      <IconA height={40} width={40} style={tailwind('relative z-10')} />
-      <IconB height={40} width={40} style={tailwind('-ml-3.5 mr-2')} />
+      <IconA height={36} width={36} style={tailwind('relative z-10')} />
```

Comment:
Would you cause a regression to other screen that is also using this component?

Reply:
Let me create a separate component for that

Reply:
i think we can just add some props